packages/manager/modules/billing/src/autoRenew/actions/terminate-all-dom/routing.js

```diff
@@ -15,15 +16,38 @@ export default /* @ngInject */ ($stateProvider) => {
         $transition$.params().serviceId,
       serviceType: /* @ngInject */ ($transition$) =>
         $transition$.params().serviceType,
-      domains: /* @ngInject */ ($http, serviceId) =>
-        $http.get(`/allDom/${serviceId}/domain`).then(({ data }) =>
-          map(data, (domain) => {
-            return {
-              name: domain,
-              selected: false,
-            };
-          }),
-        ),
+      domains: /* @ngInject */ ($http, $q, $translate, serviceId) =>
+        $http.get(`/allDom/${serviceId}/domain`).then(({ data }) => {
+          const promises = [];
+          data.forEach((domain) =>
```

Comment:
I know it does the same thing but `.map` looks a lot cleaner here to return array of promises